provision/cluster/cluster.go

```diff
@@ -23,49 +24,69 @@ type ClusterProvisioner interface {
 	ClusterHelp() provTypes.ClusterHelpInfo
 }
 
+type CreateClusterProvisioner interface {
```

Comment:
I think `ClusterProvisioner` is more clear (since we can update and delete cluster, `CreateClusterProvisioner` seems a bit off).

Reply:
There's already a `ClusterProvisioner` so I changed it to `ProvisionClusterProvisioner`